projects/aca-folder-rules/src/lib/manage-rules/manage-rules.smart-component.ts

```diff
@@ -64,8 +65,10 @@ export class ManageRulesSmartComponent implements OnInit, OnDestroy {
 
   actionsLoading$: Observable<boolean>;
   actionDefinitions$: Observable<ActionDefinitionTransformed[]>;
+  parameterConstraints$: Observable<ActionParameterConstraint[]>;
 
   private destroyed$ = new Subject<void>();
+  private _actionDefinitionsSub;
```

Comment:
Can we define type of this variable upfront?

Reply:
Added, thanks.